docs/source/cli/slide-out.rst

```diff
@@ -6,10 +6,15 @@ slide-out
 
 .. code-block:: shell
 
+    git machete slide-out --all-merged [--delete]
     git machete slide-out [-d|--down-fork-point=<down-fork-point-commit>] [--delete] [-M|--merge] [-n|--no-edit-merge|--no-interactive-rebase] [<branch> [<branch> [<branch> ...]]]
 
-Removes the given branch (or multiple branches) from the branch layout. If no branch has been specified, current branch is removed.
-Then synchronizes the downstream (child) branches of the last specified branch on the top of the upstream (parent) branch of the first specified branch.
+Removes the given branch (or multiple branches) from the branch layout.
+If no branch has been specified, current branch is slid out.
+If ``--all-merged`` is specified, all branches that have been removed from the remote are slid out instead.
```

Comment:
Actually I've rethought what this flag actually does... @raylu how about I rename it to `--removed-from-remote`? Verbose... but `--(all-)merged` evokes `git branch --merged` to me, which lists branches merged commit-wise to HEAD 🤔 which is very different from what happens here :/

Reply:
sure, sounds good. I had similar thoughts but couldn't come up with anything succinct enough